x-pack/plugins/triggers_actions_ui/server/data/lib/time_series_query.ts

```diff
@@ -147,15 +147,15 @@ export async function timeSeriesQuery(
   return getResultFromEs(isCountAgg, isGroupAgg, esResult);
 }
 
-function getResultFromEs(
+export function getResultFromEs(
   isCountAgg: boolean,
   isGroupAgg: boolean,
   esResult: estypes.SearchResponse<unknown>
 ): TimeSeriesResult {
   const aggregations = esResult?.aggregations || {};
 
   // add a fake 'all documents' group aggregation, if a group aggregation wasn't used
-  if (!isGroupAgg) {
+  if (!isGroupAgg && aggregations.dateAgg) {
     const dateAgg = aggregations.dateAgg || {};
```

Comment:
You probably don't need the `|| {}` with this change

Reply:
Removed!